.github/workflows/test_openvino_slow.yml

```diff
@@ -0,0 +1,76 @@
+name: OpenVINO - Slow Test
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "41 1 * * *" # run every day at 1:41
+  push:
+    branches:
+      - v*-release
+  pull_request:
+    types:
+      - opened
+      - labeled
+      - reopened
+      - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    if: ${{ (github.event_name == 'workflow_dispatch') || (github.event_name == 'schedule') ||  (github.event_name == 'push') || contains( github.event.pull_request.labels.*.name, 'openvino-test') }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-22.04", "windows-2019"]
+        openvino-version: ["stable", "nightly"]
+        transformers-version: ["4.36.0", "latest"]
+        nncf: ["nncf", "git+https://github.com/openvinotoolkit/nncf.git"]
```

Comment:
To be honest, I don't think all these 16 combinations are strictly necessary. 

It also so happens that I've just started working on introducing a separate OV workflow to run not only slow OV tests, but all OV tests: https://github.com/huggingface/optimum-intel/pull/985. This is because during slow tests almost none quantization-related tests are actually run which is especially critical for ensuring nncf develop compatibility.

So after this PR is merged, if there are no objections I would remove ov-nightly and nncf-develop from slow tests in my PR and introduce a separate workflow to test ov-nightly and nncf-develop versions on full OV scope on a single OS and transformers version.